### ISSUE TITLE
/eth/v1/validator/prepare_beacon_proposer returns 404

### ISSUE THREAD

**Issue by mcdee** [status-im/nim-beacon-chain#3695]
The beacon node API endpoint `/eth/v1/validator/prepare_beacon_proposer` is not available in the nimbus beacon node.  this should be present as per the spec information at https://ethereum.github.io/beacon-APIs/#/ValidatorRequiredApi/prepareBeaconProposer

```
$ curl -s -w "%{http_code}\n" -H 'Content-type: application/json' --data '[{"validator_index": "1","fee_recipient": "0xabcf8e0d4e9587369b2301d0790347320302cc09"}]' http://localhost:xxxx/eth/v1/validator/prepare_beacon_proposer
404
```

**Comment by mcdee** [comment]
Is there any plan to implement this prior to the prater/goerli merge?  This stops Vouch from being able to use nimbus for merged chains.

**Comment by zah** [comment]
This have been implemented for the 22.7.0 release, but our implementation had a spec violation that precluded actual usage. The issue is tracked in https://github.com/status-im/nimbus-eth2/issues/3920 and we expect the fix to be shipped in 22.8.0
